docs/pages/filterable-list-control-panels.md

```diff
@@ -9,22 +9,150 @@ description: The filter control panel allows users to whittle down number of
+                                  <path d="M16.416 10.283A7.917 7.917 0 1 1 8.5 2.366a7.916 7.916 0 0 1 7.916 7.917zm-2.958.01a.792.792 0 0 0-.792-.792H9.284V6.12a.792.792 0 1 0-1.583 0V9.5H4.32a.792.792 0 0 0 0 1.584H7.7v3.382a.792.792 0 0 0 1.583 0v-3.382h3.382a.792.792 0 0 0 .792-.791z"></path>
+                              </svg>
+                          </span>
+                          <span class="o-expandable_cue o-expandable_cue-close">
+                              <span class="u-visually-hidden-on-mobile">
+                              Hide
+                              filters
+                              </span>
+                              <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 17 20.4" class="cf-icon-svg">
+                                  <path d="M16.416 10.283A7.917 7.917 0 1 1 8.5 2.366a7.916 7.916 0 0 1 7.916 7.917zm-2.958.01a.792.792 0 0 0-.792-.792H4.32a.792.792 0 0 0 0 1.583h8.346a.792.792 0 0 0 .792-.791z"></path>
+                              </svg>
+                          </span>
+                      </span>
+                  </button>
+                  <div class="o-expandable_content o-expandable_content__onload-open">
+                      <form method="get" action="#">
+                          <div class="content-l">
+                              <div class="content-l_col
+                                  content-l_col-1">
+                                  <div class="o-form_group">
+                                      <div class="m-form-field">
+                                          <label class="a-label a-label__heading" for="o-filterable-list-controls_title">
+                                          Search by keyword(s)
+                                          </label>
+                                          <input type="text" name="title" id="o-filterable-list-controls_title" class="a-text-input a-text-input__full" maxlength="250">
+                                      </div>
+                                  </div>
+                              </div>
+                              <div class="content-l_col
+                                  content-l_col-1-3">
+                                  <div class="o-form_group">
+                                      <div class="m-form-field">
+                                          <label class="a-label a-label__heading" for="o-filterable-list-controls_categories">
+                                          Category
+                                          </label>
+                                          <div class="o-multiselect">
+                                              <ul class="o-multiselect_choices"></ul>
+                                              <header class="o-multiselect_header"><input class="o-multiselect_search a-text-input" type="text" placeholder="Select up to five" id="o-filterable-list-controls_categories" autocomplete="off" style="cursor: auto;"></header>
```

Comment:
Sorry just noticed one more, `style="cursor: auto;"` is added from the Multiselect JS and can be removed.